src/mapdata.h

```diff
@@ -447,6 +447,8 @@ struct submap {
     trap_id         trp[SEEX][SEEY];  // Trap on each square
     int             rad[SEEX][SEEY];  // Irradiation of each square
 
+    bool            known[SEEX][SEEY]; // has the player ever seen this tile
```

Comment:
Ui, this means telepathic knowledge transfer between all the players on the same world. What mutation allows that?